BPA/Ch.10/js_tempran.stan

```diff
@@ -80,103 +149,52 @@ transformed data {
   int<lower=0,upper=n_occasions> last[M];
 
   for (i in 1:M)
-    first[i] <- first_capture(y[i]);
+    first[i] = first_capture(y[i]);
   for (i in 1:M)
-    last[i] <- last_capture(y[i]);
+    last[i] = last_capture(y[i]);
 }
 
 parameters {
   real<lower=0,upper=1> mean_phi;             // Mean survival
   real<lower=0,upper=1> mean_p;               // Mean capture
   vector<lower=0,upper=1>[n_occasions] gamma; // Removal entry probability
   vector[n_occasions-1] epsilon;
-  real<lower=0> sigma;
+  real<lower=0,upper=5> sigma;
+  // In case of using a weakly informative prior
+  //  real<lower=0> sigma;
 }
 
 transformed parameters {
-  matrix<lower=0,upper=1>[M, n_occasions-1] phi;
+  matrix<lower=0,upper=1>[M, n_occasions - 1] phi;
   matrix<lower=0,upper=1>[M, n_occasions] p;
   matrix<lower=0,upper=1>[M, n_occasions] chi;
 
   // Constraints
   for (i in 1:M) {
     for (t in 1:(n_occasions - 1))
-      phi[i, t] <- inv_logit(logit(mean_phi) + epsilon[t]);
+      phi[i, t] = inv_logit(logit(mean_phi) + epsilon[t]);
     for (t in 1:n_occasions)
-      p[i, t] <- mean_p;
-  } //i
+      p[i, t] = mean_p;
+  }
 
   // Uncaptured probability
-  chi <- prob_uncaptured(M, n_occasions, p, phi);
+  chi = prob_uncaptured(M, n_occasions, p, phi);
 }
 
 model {
-  vector[n_occasions] qgamma;
-
-  qgamma <- 1.0 - gamma;
-
   // Priors
-  mean_phi ~ uniform(0, 1);
-  mean_p ~ uniform(0, 1);
-  gamma ~ uniform(0, 1);
+  // Uniform priors are implicitly defined.
+  //  mean_phi ~ uniform(0, 1);
+  //  mean_p ~ uniform(0, 1);
+  //  gamma ~ uniform(0, 1);
+  //  sigma ~ uniform(0, 5);
+  // In case of using a weakly informative prior
```

Comment:
same comment as before on this extra comment---it's not necessary in the code as it's not documenting the code itself